src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5168,4 +5168,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_FieldlikeEventsInRoStruct" xml:space="preserve">
     <value>Field-like events are not allowed in readonly structs.</value>
   </data>
+  <data name="IDS_FeatureRefExtensionMethods" xml:space="preserve">
+    <value>reference extension methods</value>
```

Comment:
Will replace with "ref or in extension methods". Can't come up with anything better.

---
In reply to: [124693132](https://github.com/dotnet/roslyn/pull/20403#discussion_r124693132) [](ancestors = 124693132,123672332)